templates/java/build-alt.gradle.mustache

```diff
@@ -94,7 +94,7 @@ if (rootProject.hasProperty('mavenRepoUrl')) {
     }
     repository(url: mavenRepoUrl, configureAuth)
     pom.project {
-      name "io.gapi:grpc-google-pubsub-v1"
+      name "com.google.api:grpc-{{api.name}}-{{api.version}}"
```

Comment:
nit: I recommend using `{{{api.name}}}` (three braces). This will prevent mustache from doing any HTML encoding when substituting, which presumably is unwanted here.